packages/react-shared-contexts/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "outDir": "lib",
+    "module": "commonjs",
+    "lib": ["es5", "es2015.promise", "dom"],
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noImplicitAny": true,
+    "noUnusedLocals": true,
+    "strictNullChecks": true,
+    "forceConsistentCasingInFileNames": true,
+    "moduleResolution": "node",
+    "preserveConstEnums": true,
+    "skipLibCheck": true,
+    "typeRoots": ["../../node_modules/@types", "../../typings"],
+    "types": ["jest", "custom-global", "node"]
```

You can also remove the jest reference here if you decide to remove test stuff
```suggestion
    "typeRoots": ["../../node_modules/@types", "../../typings"],
    "types": ["jest", "custom-global"]
```

---

Done!